gcloud-java-logging/baseline/src/main/java/com/google/gcloud/logging/spi/v2/MetricsServiceV2Settings.java

```diff
@@ -95,181 +94,68 @@
-    this.methods = methods;
+  public SimpleCallSettings<UpdateLogMetricRequest, LogMetric> updateLogMetricSettings() {
+    return updateLogMetricSettings;
   }
 
-  /**
-   * Returns the PageStreamingApiCallableBuilder for the API method listLogMetrics.
-   *
-   * <!-- manual edit -->
-   * <!-- end manual edit -->
-   */
-  public PageStreamingApiCallableBuilder<ListLogMetricsRequest, ListLogMetricsResponse, LogMetric>
-      listLogMetricsMethod() {
-    return methods.listLogMetricsMethod;
+  public SimpleCallSettings<DeleteLogMetricRequest, Empty> deleteLogMetricSettings() {
+    return deleteLogMetricSettings;
   }
 
-  /**
-   * Returns the ApiCallableBuilder for the API method getLogMetric.
-   *
-   * <!-- manual edit -->
-   * <!-- end manual edit -->
-   */
-  public ApiCallableBuilder<GetLogMetricRequest, LogMetric> getLogMetricMethod() {
-    return methods.getLogMetricMethod;
+  public static MetricsServiceV2Settings defaultInstance() throws IOException {
+    return newBuilder().build();
   }
 
-  /**
-   * Returns the ApiCallableBuilder for the API method createLogMetric.
-   *
-   * <!-- manual edit -->
-   * <!-- end manual edit -->
-   */
-  public ApiCallableBuilder<CreateLogMetricRequest, LogMetric> createLogMetricMethod() {
-    return methods.createLogMetricMethod;
+  public static Builder newBuilder() {
+    return new Builder();
   }
 
-  /**
-   * Returns the ApiCallableBuilder for the API method updateLogMetric.
-   *
-   * <!-- manual edit -->
-   * <!-- end manual edit -->
-   */
-  public ApiCallableBuilder<UpdateLogMetricRequest, LogMetric> updateLogMetricMethod() {
-    return methods.updateLogMetricMethod;
+  public Builder toBuilder() {
+    return new Builder(this);
   }
 
-  /**
-   * Returns the ApiCallableBuilder for the API method deleteLogMetric.
-   *
-   * <!-- manual edit -->
-   * <!-- end manual edit -->
-   */
-  public ApiCallableBuilder<DeleteLogMetricRequest, Empty> deleteLogMetricMethod() {
-    return methods.deleteLogMetricMethod;
+  private MetricsServiceV2Settings(Builder settingsBuilder) throws IOException {
+    super(
+        settingsBuilder.getOrBuildChannel(),
+        settingsBuilder.shouldAutoCloseChannel(),
+        settingsBuilder.getOrBuildExecutor(),
+        settingsBuilder.getConnectionSettings(),
+        settingsBuilder.getGeneratorName(),
+        settingsBuilder.getGeneratorVersion(),
+        settingsBuilder.getClientLibName(),
+        settingsBuilder.getClientLibVersion());
+
+    listLogMetricsSettings = settingsBuilder.listLogMetricsSettings().build();
+    getLogMetricSettings = settingsBuilder.getLogMetricSettings().build();
+    createLogMetricSettings = settingsBuilder.createLogMetricSettings().build();
+    updateLogMetricSettings = settingsBuilder.updateLogMetricSettings().build();
+    deleteLogMetricSettings = settingsBuilder.deleteLogMetricSettings().build();
   }
 
-  private static PageDescriptor<ListLogMetricsRequest, ListLogMetricsResponse, LogMetric>
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/mziccard/gcloud-java/file/2388714612/issues/source?bid=3123104&fileBranchId=3228843#l156)